lib/src/chocolatey.dart

```diff
@@ -87,6 +88,21 @@ String get chocolateyDartVersion {
   return result.toString();
 }
 
+/// The Chocolatey Repo Url
+///
+/// This is used for the Chocolatey verification file.
+/// It defaults to [githubRepo].
+final chocolateyRepoUrl = InternalConfigVariable.fn<String>(
+  () => 'https://github.com/${githubRepo.value}',
```

Comment:
If the user hasn't set `chocolateyRepoUrl`, they may be confused by the fact that they get an error about GitHub even if they aren't using GitHub deployment at all. It's probably better to explicitly fail with a message that says `"pkg.chocolateyRepoUrl or pkg.githubRepo must be set to deploy to Chocolatey."` Same for `chocolateyReleaseUrl`.

Reply:
@nex3 I have limited time, but I would love to finish this out over this weekend. Do you mind pointing me at some examples of these tests as I am still becoming familiar with the inner workings of the package?

Reply:
The GitHub configuration tests are a pretty good place to look: https://github.com/google/dart_cli_pkg/blob/bf2815ac4c554c2fbda0091d788ef0c43bb19999/test/github_test.dart#L41